MuonAnalysis/MuonAssociators/interface/PropagateToMuonCore.h

```diff
@@ -1,15 +1,12 @@
-#ifndef MuonAnalysis_MuonAssociators_interface_PropagateToMuon_h
-#define MuonAnalysis_MuonAssociators_interface_PropagateToMuon_h
+#ifndef MuonAnalysis_MuonAssociators_interface_PropagateToMuonCore_h
+#define MuonAnalysis_MuonAssociators_interface_PropagateToMuonCore_h
 //
 //
 
 /**
-  \class    PropagateToMuon PropagateToMuon.h "MuonAnalysis/MuonAssociators/interface/PropagateToMuon.h"
-  \brief    Propagate an object (usually a track) to the second muon station.
-            Support for other muon stations will be added on request.
-            
-  \author   Giovanni Petrucciani
-  \version  $Id: PropagateToMuon.h,v 1.3 2010/07/01 07:41:09 gpetrucc Exp $
+  \class    PropagateToMuonCore PropagateToMuonCore.h
+  "MuonAnalysis/MuonAssociators/interface/PropagateToMuonCore.h" \brief    
+ Propagate an object (usually a track) to the second (default) or first muon station.
```

Comment:
Spurious text (should just be `  \brief  [text]`).

Reply:
Done (if I understood correctly)